app/controllers/events_controller.rb

```diff
@@ -35,6 +35,8 @@ def invalid_request(message = 'invalid request', type: 'invalid_request')
   end
 
   def verify_signature
+    return true if Rails.env.development?
```

Comment:
This is why I added https://github.com/rainforestapp/rainforest-integrations/pull/21/files#diff-69b2650604ff734e515c98c3fba94314R161

Maybe it's not complete enough? There are already tests making sure it's doing its job in the `events_controller_spec`. Let me know if that's enough or not.